rllib/policy/tf_policy.py

```diff
@@ -1187,11 +1187,12 @@ def _get_loss_inputs_dict(self, train_batch: SampleBatch, shuffle: bool):
         # Build the feed dict from the batch.
         feed_dict = {}
         for key, placeholders in self._loss_input_dict.items():
-            tree.map_structure(
+            a = tree.map_structure(
                 lambda ph, v: feed_dict.__setitem__(ph, v),
                 placeholders,
                 train_batch[key],
             )
+            del a
```

Comment:
is this a memory leak, do we need to do this everywhere we use tree.map_structure?